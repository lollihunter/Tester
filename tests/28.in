2 20 20
1 1
2 15
2 19
2 2
2 9
1 9
1 14
1 7
1 4
1 19
1 5
2 16
2 3
2 10
1 6
1 2
2 18
2 11
2 1
2 20
