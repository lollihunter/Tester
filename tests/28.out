15
1 16 9 11 15 8 5 14 18 7 2 12 17 3 20 
